ODLA/platforms/vodla/include/vodh_client/vodh_client.cpp

```diff
@@ -0,0 +1,86 @@
+#include <fstream>
```

Comment:
why this file is placed under "include" directory?
how about ODLA/platforms/vodla/vodh_client/?

what's the purpose of this file? for testing?